.github/workflows/push-pypi.yml

```diff
@@ -41,17 +41,6 @@ jobs:
       - name: Check the archives
```

Comment:
I am also dropping the push to testPYPI here. it would be better for us to implement pre-releases then use test pypi. pydantic is not maintained on testpypi so nothing would ever actually work as we want it to if we test from test pypi. also we could open users to vulnerabilities because if a dependency of ours is uploaded to testpypi by a "bad actor" we might inadvertently install that bad package in our dependency suite locally .